pkg/kubelet/server/server_test.go

```diff
@@ -1635,3 +1640,30 @@ func TestCRIHandler(t *testing.T) {
 	assert.Equal(t, path, fw.criHandler.RequestReceived.URL.Path)
 	assert.Equal(t, query, fw.criHandler.RequestReceived.URL.RawQuery)
 }
+
+func TestDebuggingDisabledHandlers(t *testing.T) {
+	fw := newServerTest(false)
+	defer fw.testHTTPServer.Close()
+
+	paths := []string{
+		"/run", "/exec", "/attach", "/portForward", "/containerLogs", "/runningpods",
+		"/run/", "/exec/", "/attach/", "/portForward/", "/containerLogs/", "/runningpods/",
+		"/run/xxx", "/exec/xxx", "/attach/xxx", "/debug/pprof/profile", "/logs/kubelet.log",
+	}
+
+	for _, p := range paths {
+		resp, err := http.Get(fw.testHTTPServer.URL + p)
+		require.NoError(t, err)
+		assert.Equal(t, http.StatusMethodNotAllowed, resp.StatusCode)
+		body, err := ioutil.ReadAll(resp.Body)
+		require.NoError(t, err)
+		assert.Equal(t, "Debug endpoints are disabled.\n", string(body))
+
+		resp, err = http.Post(fw.testHTTPServer.URL+p, "", nil)
+		require.NoError(t, err)
+		assert.Equal(t, http.StatusMethodNotAllowed, resp.StatusCode)
+		body, err = ioutil.ReadAll(resp.Body)
+		require.NoError(t, err)
+		assert.Equal(t, "Debug endpoints are disabled.\n", string(body))
+	}
```

Comment:
Would be better to test that default handlers are still accessable, when debug handlers are disabled.

Reply:
o, maybe update existing test cases with => fw := newServerTest(false) ???

Ha, this gives a good reason to use fw := newServerTest(true).

Reply:
Cant understand it.

Reply:
I think it would be sufficient to just call a couple of the other paths, and verify that you get a 200 response code.

Reply:
Done!